libs/parsers/src/lib/expression/parser.ts

```diff
@@ -132,11 +139,23 @@ export class ExpressionParser {
       if (scanner.getTokenForTypeAndValue(ExpressionTokenType.LeftSquareBracket)) {
         const expression = this._parseExpressionRecursively(scanner);
         token = ExpressionParser._getTokenOrThrowException(scanner, ExpressionTokenType.RightSquareBracket);
-        dereferences.push({
-          isSafe,
-          isDotNotation: false,
-          expression: expression,
-        });
+
+        if (expression.type === ExpressionType.StringLiteral) {
+          // takes care of expressions that are nested such as ['body/value']
+          for (const expressionValue of (expression as ExpressionLiteral).value.split('/')) {
```

Comment:
You cannot assume here / means it is openApiConnection aliasing format, what if properties actually have this character, so please add some kind of aliasing support flag in the parser so it knows it has to assume / as open api alias parameter name

Reply:
Changed the code to this:
```
if (expression.type === ExpressionType.StringLiteral && scanner.referenceKeyFormat === ConnectionReferenceKeyFormat.OpenApi) {
          // takes care of expressions that are nested such as ['body/value']
          for (const expressionValue of (expression as ExpressionLiteral).value.split('/')) {
            dereferences.push({
              isSafe,
              isDotNotation: false,
              expression: { type: ExpressionType.StringLiteral, value: expressionValue },
            });
          }
```